.trivyignore

```diff
@@ -7,6 +7,28 @@
 # To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
 # warning at the top of this file.
 #
+# Suppression for snakeyaml 1.30 vulnerability as bundled with application insights so can't be upgraded easily
```

Comment:
@dhirajshettymoj @adaviesMOJ should I just directly edit this PR and put it back to how it was?

Reply:
I thought this was auto updated, do we know what causes this change? As it might be needed?

Reply:
Updated to the correct version after doing a gradle build